src/components/LibraryItem.tsx

```diff
@@ -188,9 +188,10 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     }
 
     onLibraryItemClicked() {
-
-        // Toggle expansion state.
         let currentlyExpanded = this.state.expanded;
         this.setState({ expanded: !currentlyExpanded });
+
+        let libraryView = this.props.libraryView;
+        libraryView.raiseEvent("itemClicked", this);
```

Comment:
Change it to `libraryView.raiseEvent("itemClicked", this.item.props.data. creationName);`